moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp

```diff
@@ -370,35 +373,96 @@ class Ros2ControlManager : public moveit_controller_manager::MoveItControllerMan
     return c;
   }
 
+  static void simplifyControllerActivationDeactivation(std::vector<std::string>& activate_controllers,
```

Comment:
Why is this a static method and not just a function?

Reply:
Moved to function